src/components/organisms/RichMenu/RichMenuDescriptionContent/RichMenuDescriptionContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "@components/atoms";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/outline";
+import { RichMenuDescriptionContentProps } from "./type";
+
+export const RichMenuDescriptionContent = ({
+  headingText,
+  description,
+  imgSrc,
+}: RichMenuDescriptionContentProps) => {
+  return (
+    <div className="m-4 flex max-h-[400px] w-[544px] flex-col px-6 py-4">
+      <div className="flex flex-col items-center gap-y-4">
+        <h2 className="text-lg font-bold leading-tight">{headingText}</h2>
+        <p className="text-xs leading-normal">{description}</p>
+      </div>
+      <img
+        src={imgSrc}
+        className="mt-6 flex-1 overflow-hidden rounded-lg object-cover"
+        draggable={false}
+      />
```

Comment:
An image should always have an `alt` attribute, even if it is empty.
```suggestion
      <img
        src={imgSrc}
        className="mt-6 flex-1 overflow-hidden rounded-lg object-cover"
        draggable={false}
        alt=""
      />
```